pkg/cmd/cli/chassis/boot/show.go

```diff
@@ -87,7 +88,11 @@ func getBootInformation(host string) interface{} {
 		}
 
 		for _, b := range systems[0].Boot.BootOrder {
-			ep := fmt.Sprintf("%s/%s", bo, strings.TrimPrefix(b, "Boot"))
+			// trim "Boot" and preceding zeros if they exist on the string
+			ep := fmt.Sprintf("%s/%s", bo, strings.TrimLeft(strings.TrimPrefix(b, "Boot"), "0"))
```

Comment:
Let's keep the zeroes.

Reply:
assuming this is how it'll print to the user.

Reply:
I think this was part of the bug...